Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM node:8.9.1
```

Comment:
ohh is this just temporary? or are we still going to try to set up docker with all the dependencies in another task?

Reply:
That's a good point, and this should be pointing at 8.9.0 anyways